src/main.rs

```diff
@@ -16,6 +18,43 @@ mod tests;
 #[macro_use]
 extern crate tracing;
 
+#[derive(Parser, Clone, Debug)]
+struct MailgunConfig {
+    /// Url used by mailgun
+    #[clap(long = "mailgun-url", env = "MAILGUN_URL")]
+    url: String,
+    /// Username used by mailgun
+    #[clap(long = "mailgun-user", env = "MAILGUN_USER")]
+    user: String,
+    /// Password used by mailgun
+    #[clap(long = "mailgun-password", env = "MAILGUN_PASSWORD")]
+    password: String,
+    /// Initial To-value used by mailgun
+    #[clap(long = "mailgun-to", env = "MAILGUN_TO")]
+    to: String,
+    /// From-value used by mailgun
+    #[clap(long = "mailgun-from", env = "MAILGUN_FROM")]
+    from: String,
+}
+
+#[derive(Parser, Clone, Debug)]
+#[command(version, about, long_about = None)]
+struct LaskugenConfig {
+    #[clap(flatten)]
+    mailgun: MailgunConfig,
+    /// The listen port for the HTTP server
+    #[clap(long, env, required = false, default_value = "3000")]
+    port: u16,
+    /// The ip address to bound by the HTTP server
+    #[clap(long, env, required = false, default_value = "127.0.0.1")]
+    bind_addr: std::net::IpAddr,
```

Comment:
I guess this could be set to 0.0.0.0 in the `Dockerfile`, as that is probably the default you want to use with containers.